lib/diagnostic.js

```diff
@@ -48,6 +48,7 @@ let scopes = [/* your response here */];
 //
 let fireOnAlderaan = function () {
   // your code here
+  deathStar.fire.call(alderaan);
```

Comment:
You need to return this in order for tests to pass.
`return deathStar.fire.call(alderaan);`